.github/workflows/ci.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: Install dependencies
         run: |
-          brew update
+          #brew update
```

Comment:
Build was failing on dependencies. brew update would reinstall a lot of things and wastes a lot of time.

Commenting it out make the sitl build for mac much faster